Configuration/NodeTypes.Document.Page.yaml

```diff
@@ -0,0 +1,54 @@
+##
+# A new Document NodeType "Page"
+#
+'Neos.Demo:Document.Page':
+  superTypes:
+    'Neos.Neos:Document': true
+  ui:
+    label: i18n
+    icon: 'file'
+    help:
+      message: 'A Page'
+    inspector:
+      groups:
+        image:
+          label: 'Neos.Demo:NodeTypes.Page:groups.image'
```

Comment:
The labels don't resolve as the folders are called differently, but changing them didn't fix it for me and I'm not quite sure why.